docassemble/__init__.py

```diff
@@ -1,5 +1,2 @@
-try:
-    __import__('pkg_resources').declare_namespace(__name__)
-except ImportError:
-    __path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
I'm ready to merge this other than this change which popped itself in here for some reason. Does anyone know why it did that? Should this be changed back?

Reply:
Seems like an upstream Docassemble change which is likely fine, unless we had to intentionally change from the Docassemble defaults in an earlier commit

Reply:
I'm guessing the original behavior was legacy for python 2.7.x